### ISSUE TITLE
Asking for 'truth' value on 'resnet-v2-101' model

### ISSUE THREAD

**Issue by sehyun03** [Microsoft/MMdnn#369]
Hi thanks for great work.

I changed "resnet-v2-101 ckpt file" on tensorflow/slim into pytorch codes.
Following https://github.com/Microsoft/MMdnn/issues/22

When I check this 'pth' weight file  on pytorch imagenet classification tasks,
it didn't worked. (imagenet frozen cls test gets no accuracy at all)

Therefore I want to check if the weight file created properly by your "imagenet_test/TestKit/truth" value, which doesn't including resnet_v2_101.

Platform (like ubuntu 16.04/win10):
ubuntu 16.04
Python version:
2.7
Source framework with version (like Tensorflow 1.4.1 with GPU):
 tensorflow 1.10.0
Destination framework with version (like CNTK 2.3 with GPU):
pytorch 0.3.0post4
Pre-trained model path (webpath or webdisk path):
https://github.com/tensorflow/models/tree/master/research/slim/resnet_v2_101_2017_04_14.tar.gz
Running scripts:
python -m mmdnn.conversion.examples.tensorflow.extract_model -n resnet_v2_101 -ckpt resnet_v2_101.ckpt
python -m mmdnn.conversion._script.convertToIR -f tensorflow -d kit_imagenet -n imagenet_resnet_v2_101.ckpt.meta --dstNodeName Squeeze -w imagenet_resnet_v2_101.ckpt
 python -m mmdnn.conversion._script.IRToCode -f pytorch --IRModelPath kit_imagenet.pb --dstModelPath kit_imagenet.py --IRWeightPath kit_imagenet.npy -dw kit_pytorch.npy
python -m mmdnn.conversion.examples.pytorch.imagenet_test --dump resnet.pth -n kit_imagenet.py -w kit_pytorch.npy

**Comment by sehyun03** [comment]
Thank you @namizzz ,
I think I should check the input or preprocess.
Again, thank you for verifying for me.
